tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -0,0 +1,68 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const fetch = require('node-fetch')
+require('url-search-params-polyfill')
+const httpHelper = require('../helpers/httpHelper')
+const assert = require('assert')
+
+When(
+  'the user creates a new public link for file/folder/resource {string} using the webUI',
+  function (resource) {
+    return client.page.FilesPageElement
+      .filesList()
+      .closeSidebar(100)
+      .openPublicLinkDialog(resource)
+      .addNewLink()
+  }
+)
+
+When('the public uses the webUI to access the last public link created by user {string}', async function (linkCreator) {
+  const headers = httpHelper.createAuthHeader(linkCreator)
+  const apiURL = client.globals.backend_url + '/ocs/v2.php/apps/files_sharing/api/v1/shares?format=json'
+  let lastShareToken = null
+  await fetch(apiURL, { method: 'GET', headers: headers })
+    .then(res => res.json())
+    .then(function (sharesResult) {
+      if (sharesResult.ocs.meta.statuscode === 200) {
+        const shares = sharesResult.ocs.data
+        let lastFoundShareId = 0
+        for (var shareI = 0; shareI < shares.length; shareI++) {
+          const share = shares[shareI]
+          if (share.share_type === 3 && share.id >= lastFoundShareId) {
```

Comment:
sharing helper JS knows a name for this magic number `3`.
Is there some reasonable way to use that knowledge here, rather than writing literal `3` ?